tests/testthat/test-validate_hub_config.R

```diff
@@ -37,3 +40,19 @@ test_that("Hub with config errors fails validation", {
     "v2.0.0"
   )
 })
+
+test_that("Target data files handled appropriately", {
+  hub_path <- system.file("testhubs/v6/target_file/", package = "hubUtils")
+  expect_message(
+    hub_val <- validate_hub_config(
+      hub_path = hub_path
+    ),
+    regexp = "target-data.json" # Ensure target-data.json is mentioned in message.
+  )
+  expect_s3_class(hub_val, "hubval")
+  expect_named(
+    hub_val,
+    c("tasks", "admin", "target-data", "model-metadata-schema")
+  )
+  expect_true(all(unlist(hub_val)))
+})
```

Comment:
Should we also had a test with a failing target-data validation to have all possible outcome tested?